### PR TITLE
Update messages_it_IT.properties

### PR DESCRIPTION
**Pull request by albanobattistella**
# Description

Please provide a summary of the changes, including relevant motivation and context.

Closes #(issue_number)

## Checklist

- [ ] I have read the [Contribution Guidelines](https://github.com/Stirling-Tools/Stirling-PDF/blob/main/CONTRIBUTING.md)
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] I have read the section [Add New Translation Tags](https://github.com/Stirling-Tools/Stirling-PDF/blob/main/HowToAddNewLanguage.md#add-new-translation-tags) (for new translation tags only)

### CODE REVIEW

File: src/main/resources/messages_it_IT.properties [hunk 1]
```diff
@@ -551,10 +551,10 @@ login.oauth2AccessDenied=Accesso negato
 login.oauth2InvalidTokenResponse=Risposta token non valida
 login.oauth2InvalidIdToken=Id Token non valido
 login.userIsDisabled=L'utente è disattivato, l'accesso è attualmente bloccato con questo nome utente. Si prega di contattare l'amministratore.
-login.alreadyLoggedIn=You are already logged in to
-login.alreadyLoggedIn2=devices. Please log out of the devices and try again.
-login.toManySessions=You have too many active sessions
-login.toManySessions2=Please log out of the devices and try again. Alternatively, you can upgrade to Stirling PDF Pro.
+login.alreadyLoggedIn=Hai già effettuato l'accesso a
+login.alreadyLoggedIn2=dispositivi. Esci dai dispositivi e riprova.
+login.toManySessions=Hai troppe sessioni attive
```
Comment (Frooodle) [inline src/main/resources/messages_it_IT.properties:556]: Actually let me remove these messages, decided not to use this functionality

Comment (Frooodle) [reply src/main/resources/messages_it_IT.properties:556]: Will update later today and you can pull changes

Comment (albanobattistella) [reply src/main/resources/messages_it_IT.properties:556]: > Will update later today and you can pull changes

HI,

I sent an email.
thanks.
A.

Comment (Frooodle) [reply src/main/resources/messages_it_IT.properties:556]: Will merge for now and delete later :)

**Pull request merged**